src/test/java/task_40/Task40Locators.java

```diff
@@ -0,0 +1,28 @@
+package task_40;
+
+import org.openqa.selenium.By;
+
+public class Task40Locators {
+    // Test2
+    static final By.ById SELECT = new By.ById("multi-select");
+    // Test3
+    static final By.ByXPath CLICK_GO_GET_CONFIRM_BOX_BUTTON = new By.ByXPath("//*[text()='Click the button to display an confirm box:']/following-sibling::button");
+    static final By.ByXPath ACCEPT_CONFIRM_BOX_TEXT = new By.ByXPath("//*[text()='Click the button to display an confirm box:']/following-sibling::p[text()='You pressed OK!']");
+    static final By.ByXPath DISMISS_CONFIRM_BOX_TEXT = new By.ByXPath("//*[text()='Click the button to display an confirm box:']/following-sibling::p[text()='You pressed Cancel!']");
```

Comment:
Please make one By element. Use getText() to get different text according to the action on the page